composer.json

```diff
@@ -5,8 +5,8 @@
     "description": "A pack for the Doctrine ORM",
     "require": {
         "composer/package-versions-deprecated": "*",
-        "doctrine/orm": "^2",
-        "doctrine/doctrine-bundle": "^2",
-        "doctrine/doctrine-migrations-bundle": "^2"
+        "doctrine/orm": "*",
+        "doctrine/doctrine-bundle": "*",
+        "doctrine/doctrine-migrations-bundle": "*"
```

Comment:
please use `^3`

Reply:
I agree with the above comment - why not make orm-pack 2.0 stick to migrations 3.x only ? It seems that we are chasing again the same problems we got when migrations-bundle got bumped from 2 to 3...